odoo/addons/base/static/src/js/res_config_settings.js

```diff
@@ -320,17 +320,19 @@ var BaseSettingRenderer = FormRenderer.extend({
             module.settingView.find('h2').addClass('o_hidden');
             module.settingView.find('.settingSearchHeader').addClass('o_hidden');
             module.settingView.find('.o_settings_container').removeClass('mt16');
-            var resultSetting = module.settingView.find(".o_form_label:containsTextLike('" + self.searchText + "')");
-            if (resultSetting.length > 0) {
-                resultSetting.each(function () {
-                    var settingBox = $(this).closest('.o_setting_box');
+
+            const resultSetting = module.settingView.find(".o_form_label")
+                .filter((_, e) => e.innerText.toUpperCase().includes(self.searchText.toUpperCase()));
```

Comment:
Maybe compute `self.searchText.toUpperCase()` just once at the toplevel and use that in the filter & pass that to `wordHighlighter`? Recomputing it twice per label seems a bit much,